### PR TITLE
Fix crash on unhooking function when capture_data is not available

### PR DESCRIPTION
**Pull request by dimitry-**
Test: Start Orbit, load preset, unhook a function,
      make sure there is no crash.
Bug: http://b/179787649

### CODE REVIEW

**Review by ronaldfw — APPROVED** [review]
LGTM, thanks!

**Review by karupayun — APPROVED** [review]
LGTM!

File: src/OrbitGl/App.cpp [hunk 1]
```diff
@@ -1743,7 +1743,11 @@ void OrbitApp::AddFrameTrack(uint64_t instrumented_function_id) {
 }
 
 void OrbitApp::RemoveFrameTrack(const FunctionInfo& function) {
-  CHECK(HasCaptureData());
```
Comment (florian-kuebler) [inline src/OrbitGl/App.cpp:1746]: Sorry for being later. I'd argue to keep the CHECK and move the if to the caller. RemoveFrameTrack should remove an existing frametrack from the ui. It does not make sense to call it if there is not CaptureData, thus no existing frametrack.

Comment (dimitry-) [reply src/OrbitGl/App.cpp:1746]: true - I will do it in main branch... I would keep it as is here

Comment (dimitry-) [reply src/OrbitGl/App.cpp:1746]: Actually it is used in 5 different places. I think it is better if we just ignore this here. Note that this call is also ignored if the function was not previously instrumented.

**Pull request merged**